src/ProgramBlockEditor.js

```diff
@@ -150,78 +150,89 @@ class ProgramBlockEditor extends React.Component<ProgramBlockEditorProps, Progra
         switch(command) {
             case 'forward':
                 return (
-                    <Button
+                    <AriaDisablingButton
                         ref={ (element) => this.setCommandBlockRef(programStepNumber, element) }
                         key={`${programStepNumber}-forward`}
-                        data-stepnumber={programStepNumber}
                         className={classNames.join(' ')}
                         variant='command-block--forward'
+                        data-stepnumber={programStepNumber}
                         aria-label={
                             this.addIsSelected() ?
                             `${this.props.intl.formatMessage({id:'ProgramBlockEditor.commandForward'}, {index: programStepNumber + 1})}. ${this.props.intl.formatMessage({id:'ProgramBlockEditor.commandOnAdd'})}` :
                             this.deleteIsSelected() ?
                             `${this.props.intl.formatMessage({id:'ProgramBlockEditor.commandForward'}, {index: programStepNumber + 1})}. ${this.props.intl.formatMessage({id:'ProgramBlockEditor.commandOnDelete'})}` :
                             this.props.intl.formatMessage({id:'ProgramBlockEditor.commandForward'}, {index: programStepNumber + 1})
                         }
-                        onClick={this.handleClickStep}>
+                        disabledClassName='ProgramBlockEditor__program-block--disabled'
```

Comment:
Please make `disabledClassName` optional on `AriaDisablingButton` and remove the attribute from the program blocks. We want to set `aira-disabled` on the program blocks, but we don't need any presentation change.